google/cloud/bigtable/tools/convert_acceptance_tests.py

```diff
@@ -24,16 +24,16 @@
 parser.
 
 Usage:
-
-    curl -L 'https://raw.githubusercontent.com/GoogleCloudPlatform/cloud-bigtable-client/master/bigtable-client-core-parent/bigtable-client-core/src/test/resources/com/google/cloud/bigtable/grpc/scanner/read-rows-acceptance-test.json' \
+    curl -L 'https://raw.githubusercontent.com/googleapis/conformance-tests/master/bigtable/v2/readrows.json' \
       | python ../tools/convert_acceptance_tests.py \
       | clang-format >readrowsparser_acceptance_tests.inc
+
 """
 from __future__ import print_function
 
 import json
 import sys
-
+import base64
```

Comment:
do we have to take care of decoding in python script?

I am skeptical about it.